osmomath/decimal.go

```diff
@@ -248,6 +248,15 @@ func (d BigDec) BigInt() *big.Int {
 	return cp.Set(d.i)
 }
 
+// BigIntMut converts BigDec to big.Int, mutative the input
+func (d BigDec) BigIntMut() *big.Int {
```

Comment:
I think ToBigInt is not a suitable method name here, recommending change to the following:
```suggestion
func (d BigDec) ToBigInt() *big.Int {
```